src/staff-handbook/pay-pension-and-benefits/pension.md

```diff
@@ -3,15 +3,10 @@ title: Pension
 related_order: 2
 last_reviewed_at: ""
 ---
+dxw provides a Pension Salary Exchange (PSE) scheme which is operated by Aviva. In a PSE scheme, contributions are deducted from your gross salary, resulting in a saving on tax and National Insurance Contributions (NICs). 
```

Comment:
```suggestion
dxw provides a pension salary exchange (PSE) scheme which is operated by Aviva. In a PSE scheme, contributions are deducted from your gross salary, resulting in a saving on tax and National Insurance contributions (NICs). 
```

Reply:
need to be capitals - reviewed and agreed by Pauline

Reply:
Both "Pension Salary Exchange" and "National Insurance Contributions"? gov.uk use "National Insurance contributions" [here](https://www.gov.uk/national-insurance) and [here](https://www.gov.uk/guidance/salary-sacrifice-and-the-effects-on-paye)

Neither of these are really proper nouns, so typically they wouldn't be capitalised, but it's a minor thing!

Reply:
in the FAQs document we shared that was supplied by our accountants [here](https://docs.google.com/document/d/10Dl3CbXKJEgb6edFv78fNmFE8yXX8_1yaxfRek1KlsY/edit?usp=sharing), the same way it is here [https://obr.uk/forecasts-in-depth/tax-by-tax-spend-by-spend/national-insurance-contributions-nics/](url). 
As it's in relation to the abbreviation straight after, it's capitalised. Neither of the examples above have the abbreviation following

Reply:
From the second example: "we forecast National Insurance contributions (NICs)". It's only capitalised in the heading, because headings are dependent on style